builtin/providers/azure/provider.go

```diff
@@ -39,108 +43,164 @@ func Provider() terraform.ResourceProvider {
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("AZURE_CERTIFICATE", ""),
 			},
+
+			// ARM-specific fields:
+			"arm_config_file": &schema.Schema{
```

Comment:
The current pattern (as of Terraform 0.6.7) here is to make the field accept the contents of the file, and then allow use of the `file` variable interpolation.